src/components/two-factor-authentication/SelfServiceWorkflow.vue

```diff
@@ -239,7 +252,11 @@
             >
               <v-btn
                 :block="mdAndDown"
-                class="secondary button"
+                :class="
+                  !twoFactorStore.errorCode || workflowStep == TwoFactorSteps.Verify
+                    ? 'secondary button'
+                    : 'primary button'
```

Comment:
```suggestion
                    ? 'secondary'
                    : 'primary'
```
Wir haben glaube Ich kein "button" class?